### PR TITLE
DOCSP-45004: Run Command

### PR DESCRIPTION
**Pull request by mcmorisi**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-golang/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-45004>

### Staging Links

<li><a href=https://deploy-preview-320--mongodb-docs-csharp.netlify.app/fundamentals/databases-collections/run-command>fundamentals/databases-collections/run-command</a></li><li><a href=https://deploy-preview-320--mongodb-docs-csharp.netlify.app/fundamentals/database-collection>fundamentals/database-collection</a></li>

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/fundamentals/databases-collections/run-command.txt [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+free MongoDB Atlas cluster and load the sample datasets, see the :ref:`<csharp-quickstart>`.
+
+
+Execute a Command
+-----------------
+
+To run a database command, create a ``BsonDocument`` object that specifies
+the command and pass it as a parameter to the ``RunCommand()`` or ``RunCommandAsync()`` 
+method. You can specify the type returned by the ``RunCommand()`` and ``RunCommandAsync()``
+methods by specifying the type parameter. You can use the ``BsonDocument`` type to return
```
Comment (stephmarie17) [inline source/fundamentals/databases-collections/run-command.txt:57]: [s] Suggestion for brevity and reducing repetition:
```suggestion
method. You can specify the type returned by these methods by specifying the type parameter. You can use the ``BsonDocument`` type to return
```

File: source/fundamentals/databases-collections/run-command.txt [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+       value is ``0.0``.
+
+   * - ``$clusterTime``
+     - A document that contains the signed cluster time. Cluster time is a
+       logical time used for the ordering of operations. This field only
+       applies to commands run on replica sets or sharded cluster.
+
+   * - ``operationTime``
+     - The logical time of the operation execution. This field only
+       applies to commands run on replica sets or sharded cluster.
```
Comment (stephmarie17) [inline source/fundamentals/databases-collections/run-command.txt:161]: [q] Are these meant to be singular on purpose? If not, would update:
```suggestion
       applies to commands run on replica sets or sharded clusters.

   * - ``operationTime``
     - The logical time of the operation execution. This field only
       applies to commands run on replica sets or sharded clusters.
```

**Review by stephmarie17 — APPROVED** [review]
LGTM with a few nits ✅

**Review by adelinowona — APPROVED** [review]
LGTM!

**Pull request merged**